webapi/tct-fileapi-w3c-tests/tests.full.xml

```diff
@@ -9,9 +9,9 @@
         </description>
         <specs>
           <spec>
-            <spec_assertion category="Tizen W3C API Specifications" element_name="readAsArrayBuffer" element_type="method" interface="FileReader" section="Storage" specification="File API" />
+            <spec_assertion category="Tizen W3C API Specifications" element_name="readAsArrayBuffer" element_type="method" interface="FileReader" section="Storage" specification="File API"/>
```

Comment:
Actually, I cannot filter out what is the real change because there are lots of changes ` />` to `/>`.

Are you using some tool(s) here?